server/routes/auth/oauth2-callback.js

```diff
@@ -145,11 +145,7 @@ module.exports = function(config, req, res, next) {
         );
       }
 
-      // Was this sign-in triggered from the home page?
-      if (req.session.home) {
-        delete req.session.home;
-        return res.redirect(301, path.join("/", locale));
-      }
+      
```

Comment:
Revert this change please, `git checkout master server/routes/auth/oauth2-callback.js`